debian/changelog

```diff
@@ -1,3 +1,10 @@
+docker.io (20.10.21-0ubuntu4) lunar; urgency=medium
+
+  * d/t/docker-in-lxd: Fix dep8 tests to let them copy packages into
+    container properly (LP: #2012299)
```

Comment:
Could you please mention here what you did? Added the `-n` option in the `grep-dctrl` call.

Reply:
Thanks for your advice. Done